tests/bluechi_test/bluechi_is_online.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Any, Dict, Iterator, List, Optional, Tuple, Union
+
+from bluechi_test.client import Client
+
+
+class BluechiIsOnline:
+
+    binary_name = "bluechi-is-online"
+
+    def __init__(self, client: Client) -> None:
+        self.client = client
+        self.tracked_services: Dict[str, List[str]] = dict()
+
+    def run(
+            self,
+            log_txt: str,
+            cmd: str,
+            check_result: bool,
+            expected_result: int
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        result, output = self.client.exec_run(f"{BluechiIsOnline.binary_name} {cmd}")
+        if check_result and result != expected_result:
+            raise Exception(
+                f"{log_txt} failed with {result} (expected {expected_result}): {output}"
+            )
+        return result, output
+
+    def agent_is_online(self) -> int:
+        log_txt = "Checking if agent is active."
+        cmd = "agent"
+        check_result = True
+        expected_result = 0
```

Comment:
Please put above values directly into method invocation, no need to store them into intermediate variables